feeding/src/action/MoveDirectlyToPerson.cpp

```diff
@@ -84,8 +84,7 @@ bool moveDirectlyToPerson(
           planningTimeout,
           maxNumTrials,
           getConfigurationRanker(ada),
-          velocityLimits,
-          ada::TrajectoryPostprocessType::KUNZ))
+          velocityLimits))
```

Comment:
Ditto previous nit on whether accepting the default PP type is OK with y'all.

Reply:
Ditto previous comment.

Reply:
Ditto previous groovy.